include/odp/api/spec/packet_io.h

```diff
@@ -979,6 +981,20 @@ int odp_pktio_promisc_mode(odp_pktio_t pktio);
 int odp_pktio_mac_addr(odp_pktio_t pktio, void *mac_addr, int size);
 
 /**
+ * Set the default MAC address of a packet IO interface.
+ *
```

Comment:
It needs to be mentioned here that an interface supports this function only when capa.set_op.op.mac_addr == 1

Reply:
ok